src/main/java/net/sf/marineapi/ais/util/PositioningDevice.java

```diff
@@ -21,42 +21,67 @@
 package net.sf.marineapi.ais.util;
 
 /**
- * Checks the positioning device type for validity.
+ * Provides functionality related to positioning devices.
```

Comment:
How does the changed comment make an improvement here?  As compared to the original, I find it now more abstract and vague, telling less about the purpose of this class.